blogposts/announcing-sourcegraph-3.7.md

```diff
@@ -0,0 +1,193 @@
+
+## Multi-line (`\n`) search on all branches
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+In Sourcegraph 3.5 [we introduced the ability to do a multi-line search](https://about.sourcegraph.com/blog/sourcegraph-3.5#multi-line-search-with-newline-code-classlanguage-textncode-characters) by writing `\n` in queries, however it was limited to only indexed default (e.g. `master`) branches. In 3.7 we expand this to include unindexed branches so **you can now perform multi-line searches on any branch**.
+
+For example, [find all empty if and else statements in the Go language server code](https://sourcegraph.com/search?q=repo:github%5C.com/sourcegraph/go-langserver%40*refs/heads/+lang:go+%5Cbif%7Celse%5Cb+%7B%5Cn%5Cs*%7D+count:100).
+
+```
+repo:github\.com/sourcegraph/go-langserver@*refs/heads/ lang:go \bif|else\b {\n\s*} count:100
+```
+
+## Language extensions get icons
+
+![extension registry](images/3.7-extension-icons.png "extension registry")
+
+Extensions now have the option to include icons. This has been done for language extensions and will soon include external services such as Codecov and Sentry.
+
+## Sourcegraph configuration temporary overrides in the UI
+
+In Sourcegraph 3.4 we [introduced an optional way to load Sourcegraph configuration from a file or Kubernetes config map](https://about.sourcegraph.com/blog/sourcegraph-3.4#optional-loading-of-configuration-from-the-file-system-or-kubernetes-configmap). This helped teams in which Sourcegraph is a critical piece of infrastructure and checking their Sourcegraph configuration into version control was desirable. Since then, we’ve heard feedback that it would be nice to be able to make temporary edits in the web UI and diverge from the files on disk intentionally (i.e. to try an option out before committing it).
+
+You can now set `EXTSVC_CONFIG_ALLOW_EDITS=true` and `SITE_CONFIG_ALLOW_EDITS=true` to allow the active Sourcegraph configuration to diverge from the file, thus allowing temporary edits. For complete details, see [the documentation](https://docs.sourcegraph.com/admin/config/advanced_config_file).
+
+## In case you missed it...
+
+- We learned a ton at GopherCon, and you can read about some of the awesome talks from the GopherCon live-blog!
+- We had a fruitful hackweek, with one project resulting in [improved default search on sourcegraph.com]
+- We have been exploring LSIF as a fast and precise code intelligence solution. Read about [what we’ve been learning]!
+
+## 3.7.0 Changelog
```

Comment:
I was given feedback to include all patch release changes that are bubbling into the release - this ended up looking funny with the headers. Would it be better to add a style for h4, and nest the versions one level deeper under ## Changelog?

Reply:
I updated the css.